src/picongpu/include/plugins/radiation/radFormFactor.hpp

```diff
@@ -82,6 +82,29 @@ namespace picongpu
   } // end namespace: radFormFactor_CIC_1Dy
 
 
+  namespace radFormFactor_Gauss_spherical
+  {
+    class radFormFactor
+    {
+    public:
+      HDINLINE radFormFactor(void)
+      { }
+
+      HDINLINE float_X operator()(const float_X N, const float_X omega, const vector_X observer_unit_vec) const
+      {
+
+        /* Form Factor for point-symmetric Gaussian charge distribution of N discrete electrons:
```

Comment:
Compared to `radFormFactor_CIC_3D` your comment is minimalistic and not yet complete (parameters and equations). Additionally, also write down also write down the initial charge distribution.